tiling.js

```diff
@@ -380,6 +380,15 @@ export class Space extends Array {
             this.setSpaceTopbarElementsVisible(true);
         });
 
+        // after layout is complete, remove any residual fullscreenOnLayout
+        this.signals.connect(this, "layout", () => {
+            this.getWindows().forEach(w => {
+                if (w.fullscreenOnLayout) {
+                    delete w.fullscreenOnLayout;
+                }
+            });
+        });
```

Comment:
Why is this here and not at the end of `layout`?

Reply:
Good question!

No reason really, other than using the "layout" signal (we don't use it so I though "why not?").

It probably should be in `layout` and at as a `MetaLater`.

This PR still has a few issues that I'm working though (mainly, with `kodi` on unfullscreening, it doesn't resize itself... it's more a `kodi` thing but I'd like to address).  I'll set this as a draft for now.